gateway/handle_lifecycle.go

```diff
@@ -115,7 +117,7 @@ func (gw *Handle) Setup(
 	gw.processRequestTime = gw.stats.NewStat("gateway.process_request_time", stats.TimerType)
 	gw.emptyAnonIdHeaderStat = gw.stats.NewStat("gateway.empty_anonymous_id_header", stats.CountType)
 
-	gw.now = time.Now
+	gw.now = timeutil.Now
```

Comment:
I agree but not an important ATM. I made changes to the gateway and the batch router packages, since during testing I was only exposed to those packages.